### PR TITLE
add tms comment to :angle-vector-sequence

### PR DESCRIPTION
**Pull request by k-okada**
c.f. https://github.com/jsk-ros-pkg/jsk_robot/pull/791#pullrequestreview-45324124

### CODE REVIEW

**Comment by snozawa** [comment]
In general, I'd like to use (almost) common documentation with other documentations:
https://github.com/start-jsk/rtmros_common/blob/master/hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l#L552-L568
If this PR is merged, I'll update the documentation of rtm-ros-robot-interface.l

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -401,7 +401,15 @@
    av)
   (:angle-vector-sequence
    (avs &optional (tms (list 3000)) (ctype controller-type) (start-time 0.1) &key (scale 1) (min-time 0.0) (end-coords-interpolation nil))
-   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops."
+   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops.
+- asv : list of joint angle vector [rad]
```
Comment (snozawa) [inline pr2eus/robot-interface.l]: `[rad]` -> `[deg]`??

File: pr2eus/robot-interface.l [hunk 2]
```diff
@@ -401,7 +401,15 @@
...
+- tms : list duration from previous angle-vector to next goal in [msec]
+- ctype : controller method name
+- start-time : time to start moving
+- scale : if tm is not specified, it will use 1/scale of the fastest speed
```
Comment (snozawa) [inline pr2eus/robot-interface.l]: `tm` should be for `:angle-vector`.
How about `tms`?

File: pr2eus/robot-interface.l [hunk 3]
```diff
@@ -401,7 +401,15 @@
...
+- min-time : minimum time to for time to goal
```
Comment (snozawa) [inline pr2eus/robot-interface.l]: Is this for `:angle-vector`?
How about `:angle-vector-sequence`?
And, `minimum time to for time to goal` -> `minimum time to goal` in english??

File: pr2eus/robot-interface.l [hunk 4]
```diff
@@ -401,7 +401,15 @@
    av)
   (:angle-vector-sequence
    (avs &optional (tms (list 3000)) (ctype controller-type) (start-time 0.1) &key (scale 1) (min-time 0.0) (end-coords-interpolation nil))
-   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops."
+   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops.
```
Comment (snozawa) [inline pr2eus/robot-interface.l]: `joind angle` -> `joint angles`?

File: pr2eus/robot-interface.l (hunk 1 above)
Comment (k-okada) [inline pr2eus/robot-interface.l]: I see, so have to change https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/035f676471e9ff0748f2a0199dbe930685b1c04e/pr2eus/robot-interface.l#L350 too

**Comment by k-okada** [comment]
thank you for comment, this PR is to add comment to `:angle-vector-sequence` and I just copied from `:angle-vector` so there are a lot of mistakes, I also look into https://github.com/start-jsk/rtmros_common/blob/master/hrpsys_ros_bridge/euslisp/rtm-ros-robot-interface.l#L552-L568 and try to use similar expressions.

**Comment by snozawa** [comment]
LGTM.

**Pull request merged**